### DISCUSSION TITLE
Add ability to sort on selected columns

### DISCUSSION THREAD

**Discussion by RHeynsZa** [KevinVandy/material-react-table#802] [Ideas]
I'm not sure if I'm perhaps missing this in the documentation, but it would be great if we had the ability to sort on the selected column. 

I don't mind contributing this as well, since we sort of hacked a solution together to get it to work anyway

**Comment by Skippidar** [comment]
I've found this information in documentation: https://www.material-react-table.com/docs/guides/display-columns#display-column-definition-options-prop

![CleanShot 2024-03-21 at 12 27 04@2x](https://github.com/KevinVandy/material-react-table/assets/322145/8aeb60d8-8d5e-42c0-8d17-3519cd9cdd19)

There is information about enabling actions, ordering, etc. BUT such options are missing in MRT_DisplayColumnDef type and they don't work. I can't find out if it's planned or maybe just wrong information in documentation.
